File: accidentes/AMarotoChivite-Accidentes/AMarotoChivite-Accidentes-Java/src/main/java/Ficheros/Accidentes/models/dto/ListAccidentesDto.java
package Ficheros.Accidentes.models.dto;

import org.simpleframework.xml.ElementList;
import org.simpleframework.xml.Root;

import java.util.List;

@Root(name = "listAccidentes")
public class ListAccidentesDto {
    @ElementList(name = "lista_accidente", entry = "accidente", inline = true)
    private List<AccidenteDto> myList;

    public List<AccidenteDto> getMyList() {
        return myList;
    }

    public void setMyList(List<AccidenteDto> myList) {
        this.myList = myList;
    }
}

File: accidentes/AMarotoChivite-Accidentes/AMarotoChivite-Accidentes-Kotlin/src/main/kotlin/Ficheros/Accidentes/models/dto/ListAccidentesDTO.kt
package Ficheros.Accidentes.models.dto

import org.simpleframework.xml.ElementList
import org.simpleframework.xml.Root

@Root(name = "listAccidentes")
class ListAccidentesDTO {
    @field:ElementList(name = "lista_accidente", entry = "accidente", inline = true)
    var myList: List<AccidenteDTO>? = null
}

